### PR TITLE
Add BugOptionProvider and BugFilingChoiceGroup component

### PR DESCRIPTION
**Pull request by haonliu**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Added relevant unit test for your changes. (`npm run test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`npm run precheckin`)
- [ ] Added screenshots/GIFs for UI changes.

### CODE REVIEW

File: src/bug-filing/components/bug-filing-choice-group.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            };
+        });
+    };
+
+    const onChange = (ev?: React.FormEvent<HTMLElement | HTMLInputElement>, option?: IChoiceGroupOption) => {
+        userConfigMessageCreator.setBugService(option.key);
+    };
+
+    return (
+        <ChoiceGroup
```
Comment (karanbirsingh) [inline src/bug-filing/components/bug-filing-choice-group.tsx]: Do we need a label and/or accessible name for this choice group?

Comment (haonliu) [reply src/bug-filing/components/bug-filing-choice-group.tsx]: Added aria label to the choice options.

File: src/bug-filing/components/bug-filing-choice-group.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+export const BugFilingChoiceGroup = NamedSFC<BugFilingChoiceGroupProps>('BugFilingChoiceGroup', props => {
+    const { userConfigMessageCreator } = props.deps;
+
+    const getOptions: () => IChoiceGroupOption[] = () => {
+        return props.bugFilingServices.map(service => {
+            return {
+                key: service.key,
+                text: service.displayName,
+                ariaLabel: service.displayName,
```
Comment (dbjorge) [inline src/bug-filing/components/bug-filing-choice-group.tsx]: Is an `ariaLabel` necessary here? I thought it was preferable to omit it if a proper semantic `<label for="id">` with appropriate text existed, which office fabric is intending to provide as part of their `ChoiceGroup` implementation.

Comment (haonliu) [reply src/bug-filing/components/bug-filing-choice-group.tsx]: Done.

**Pull request merged**